app/components/autocomplete/at_mention_item/at_mention_item.js

```diff
@@ -34,11 +36,23 @@ export default class AtMentionItem extends PureComponent {
             userId,
             username,
             theme,
+            isBot,
         } = this.props;
 
         const style = getStyleFromTheme(theme);
         const hasFullName = firstName.length > 0 && lastName.length > 0;
 
+        let tag = null;
+        if (isBot) {
+            tag = (
+                <FormattedText
```

Comment:
Just because all the usages and the duplication of the styles, I would move it to its own component, maybe something like `<BotTag isBot={variable}/>` if `variable` is false, return null, if not, return the Formatted text. So this way is super simple to incorporate it everywhere. Anyway, maintaining the if outside the component I think is a good option too.

Reply:
Making this change.

Reply:
@jespino Ready for re-review on this.